code/modules/research/machinery/_production.dm

```diff
@@ -182,10 +183,10 @@
 			var/datum/bank_account/our_acc = card.registered_account
 			if(our_acc.account_job && SSeconomy.get_dep_account(our_acc.account_job?.paycheck_department) == SSeconomy.get_dep_account(payment_department))
 				total_cost = 0 //We are not charging crew for printing their own supplies and equipment.
-	if(attempt_charge(src, usr, total_cost) & COMPONENT_OBJ_CANCEL_CHARGE)
+	if(attempt_charge(src, usr, total_cost) & COMPONENT_OBJ_CANCEL_CHARGE) //note that, as of the time of the writing of this comment, attempt_charge() returns false if you're a silicon, so even if a silicon is dragging a card, they (currently) can't make the card pay the tax instead of their cell. they can still get the departmental discount, though.
```

Comment:
It's not wrong, but it's a rather wordy comment that's self evident within the code. If you want to change attempt_charge to charge a silicon's pulled ID or credits, than just do that.

Reply:
the problem is that changing attempt_charge() to not always return false for silicons will probably break other things, no? is attempt_charge() only used by lathe taxes?